site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst

```diff
@@ -57,3 +52,60 @@ Install Tor
         brew services start tor
 
 This will start Tor and ensure that it is always running, even after a restart.  See the `Tor Project docs <https://2019.www.torproject.org/docs/tor-doc-osx.html.en>`_ for more details.
+
+Enable Tor Systemwide
+----------------------
+
+#. Now enable apache service:
+
+    .. code-block::
+
+        sudo launchctl load -w /System/Library/LaunchDaemons/org.apache.httpd.plist
+
+#. Enable proxy autoconfig file:
+
+    .. code-block::
+
+        sudo curl https://registry.start9labs.com/sys/proxy.pac --output /Library/WebServer/Documents/proxy.pac
+
+#. Go to system preferences:
+
+    .. figure:: /_static/images/tor/systemprefs.png
+        :width: 40%
+        :alt: System Preferences
+
+#. Click on Network:
+
+    .. figure:: /_static/images/tor/network.png
+        :width: 80%
+        :alt: Select Network
+
+#. If using WiFi, select WiFi on the left panel and click "Advanced":
```

Comment:
Maybe reword this to include ethernet, instead of as a note below?  Or is it entirely impossible to use ethernet on a mac these days?

Reply:
On the laptops it is, on the Mac Pro and the iMac it isn't. As far as adapters go they all have different names so I can't give a helpful picture here given that it would only be relevant for those with the same adapter.

Reply:
I don't think a picture is necessary, just mention ethernet is an option, but that this example uses wifi